exchanges/deribit/deribit.go

```diff
@@ -1773,20 +1754,20 @@ func (d *Deribit) SubmitCancelAllByInstrument(ctx context.Context, instrument, o
 	if includeCombos {
 		params.Set("include_combos", "true")
 	}
-	var resp int64
+	var resp *MultipleCancelResponse
 	return resp, d.SendHTTPAuthRequest(ctx, exchange.RestFutures, matchingEPL, http.MethodGet,
 		submitCancelAllByInstrument, params, &resp)
 }
 
 // SubmitCancelByLabel sends a request to cancel all user orders for the specified label
 // returns the total number of successfully cancelled orders
-func (d *Deribit) SubmitCancelByLabel(ctx context.Context, label, ccy string) (int64, error) {
+func (d *Deribit) SubmitCancelByLabel(ctx context.Context, label, ccy string) (*MultipleCancelResponse, error) {
```

Comment:
[This still required `detailed` bool](https://github.com/thrasher-corp/gocryptotrader/pull/1082#discussion_r1583915250)

Reply:
cancelByLabel doesn't have the _**detailed**_ parameter either in the test package or in the documentation. That's why I didn't add it here.

<img width="367" alt="Screenshot 2024-05-07 at 2 29 47 PM" src="https://github.com/thrasher-corp/gocryptotrader/assets/39623015/4ecb08d7-e777-48f3-8c9c-15ab072203b4">

<img width="546" alt="Screenshot 2024-05-07 at 2 31 26 PM" src="https://github.com/thrasher-corp/gocryptotrader/assets/39623015/13f89186-71a9-4ec9-a8d1-29aaaa80a901">

Reply:
It is in the link I gave you: https://docs.deribit.com/#detailed-response-for-private-cancel_all-and-private-cancel_by_label-methods 
The endpoint also responds to it:
```
Deribit websocket connection: sending message [{"jsonrpc":"2.0","id":1599612810505,"method":"private/cancel_by_label","params":{"label":"test","detailed":true}}]

Deribit websocket connection: message received: {"jsonrpc":"2.0","id":1599612810505,"result":[{"instrument_name":"BTC_USDC","currency":"BTC_USDC","result":[{"is_rebalance":false,"risk_reducing":false,"order_type":"limit","creation_timestamp":1715302998260,"order_state":"cancelled","contracts":300000.0,"average_price":0.0,"post_only":false,"last_update_timestamp":1715303041949,"filled_amount":0.0,"replaced":false,"web":false,"api":true,"mmp":false,"cancel_reason":"user_request","instrument_name":"BTC_USDC","order_id":"BTC_USDC-13133482","max_show":30.0,"time_in_force":"good_til_cancelled","direction":"buy","amount":30.0,"price":30.0,"label":"test"}],"type":"limit"}],"usIn":1715303041946338,"usOut":1715303041950029,"usDiff":3691,"testnet":true}
```

```
    deribit_test.go:1620: 
        	Error Trace:	/Users/scottg/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/deribit/deribit_test.go:1620
        	Error:      	Received unexpected error:
        	            	json: cannot unmarshal array into Go struct field wsResponse.result of type int64
        	Test:       	TestWSSubmitCancelByLabel
```

Please add it to the REST and WS endpoints